### PR TITLE
Clean up Makefile and use common makefile code

### PR DESCRIPTION
**Pull request by Brian-McM**
This commit does the following:
- Use common Makefile and remove macros and target defined in the common makefile
- Add the ORGANIZATION so pin updates will work properly

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
